### PR TITLE
Fix test app API

### PR DESCRIPTION
**Pull request by vvasiloi**
Fix and enable the API in test app.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
                 php: ["8.0"]
                 symfony: ["5.4.*"]
                 sylius: ["^1.11"]
-                node: ["10.x"]
+                node: ["12.x"]
```
Comment (GSadee) [inline .github/workflows/build.yml:27]:
```suggestion
                node: ["14.x"]
```

File: tests/Application/config/packages/security.yaml [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-    sylius.security.new_api_route: "/new-api"
+    sylius.security.shop_regex: "^/(?!%sylius_admin.path_name%|api/.*|api$|media/.*)[^/]++"
+    sylius.security.new_api_route: "/api/v2"
     sylius.security.new_api_regex: "^%sylius.security.new_api_route%"
     sylius.security.new_api_admin_route: "%sylius.security.new_api_route%/admin"
     sylius.security.new_api_admin_regex: "^%sylius.security.new_api_admin_route%"
     sylius.security.new_api_shop_route: "%sylius.security.new_api_route%/shop"
     sylius.security.new_api_shop_regex: "^%sylius.security.new_api_shop_route%"
+    sylius.security.new_api_user_account_route: "%sylius.security.new_api_shop_route%/account"
+    sylius.security.new_api_user_account_regex: "^%sylius.security.new_api_user_account_route%"
```
Comment (GSadee) [inline tests/Application/config/packages/security.yaml:11]: There should be no further need to define these parameters as there was extracted to bundles in Sylius in 1.9 version I guess

**Review by GSadee — APPROVED** [review]

File: tests/Application/config/packages/security.yaml (hunk 2 above)
Comment (vvasiloi) [reply tests/Application/config/packages/security.yaml:11]: I simply copied it from Sylius-Standard.

Comment (vvasiloi) [reply tests/Application/config/packages/security.yaml:11]: I will check if they're necessary and remove if not.

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "fix-api"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by GSadee** [comment]
Thanks, Victor! :tada:

**Pull request merged**